UPGRADE-3.x.md

```diff
@@ -4,3 +4,10 @@ UPGRADE 3.x
 ## Deprecated Admin class
 
 The `Admin` class is deprecated. Use `AbstractAdmin` instead.
+
+## Deprecated template fallback mechanism
+
+The method that gets templates in the Twig extension loaded a default template
+when the one specified could not be found. You can no longer rely on that and
```

Comment:
Linebreak, especially at 80 chars, on a `.md` file is overkill IMHO.

Most of the time, we will see the rendered result: https://github.com/greg0ire/SonataAdminBundle/blob/87fd11ca21a90e398b5d419405bf767ce5448d35/UPGRADE-3.x.md

Maybe linebreak at ending sentence would be nicer to read.